### PR TITLE
Add Oauthcontroller for login through event 360 

### LINKED ISSUE #58: Login through EVENT 360
As an: CASTNXT user  
                     I want: to login with Event360 credentials and authorise as the role I want
                    So that: I do not have to maintain separate credentials for every NXT platform.

### PR DESCRIPTION
**Pull request by m-rit**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+EVENT360_CLIENT_ID=3WdrqHNMYoZR2Oii5hYMVivC1LZFZPutfZM7dhXdNCo
+EVENT360_URL = https://event360-dev-eventnxt-152500be953e.herokuapp.com
+
+EVENT360_CLIENT_SECRET=MXNaOBUX0xRJTu1qTrRVquVXBL2y_qpsb22enUoN3Jg
```
Comment (rahulb99) [inline .env:4]: this should not be committed to Github ideally

Comment (ritmanda-tamu) [reply .env:4]: yeah, I'll update it in later PRs

**Review by ritmanda-tamu — APPROVED** [review]

File: .env (hunk 1 above)
Comment (m-rit) [reply .env:4]: this pair is not permanent, separate pair will be generated for that.

**Review by JSalazar026 — APPROVED** [review]
Once the feature is finalized, this PR should be updated

File: .env (hunk 1 above)
Comment (rahulb99) [reply .env:4]: ok. Also it should be `EVENT360_URL=https://event360-dev-eventnxt-152500be953e.herokuapp.com` right?

**Pull request merged**